src/templates/layouts/newsStory/index.tsx

```diff
@@ -20,9 +21,16 @@ export const NewsStory = ({
   listing,
   lovellVariant,
   lovellSwitchPath,
+  entityPath,
+  breadcrumbs,
 }: LovellExpandedResourceTypeType<NewsStoryType>) => {
   return (
     <>
+      <Breadcrumbs
```

Comment:
I think we should include breadcrumbs at the `[[...slug]].tsx` level instead of on every page layout. It will keep things more consistent in the end, imo. 

```
      <Breadcrumbs
        breadcrumbs={resource.breadcrumbs}
        entityPath={resource.entityPath}
        hideHomeBreadcrumb={true}
      />
```

Reply:
i considered this, but different templates will need to pass in different flags.  For example, some templates hide the home breadcrumb and others dont.

Reply:
I hear you, but I think that logic can still be handled at the slug level.

also coinciding with the changes in the skip link PR, I believe we will want breadcrumbs placed outside of the `<main>` container, like va.gov is doing:
![Screenshot 2023-10-12 at 11 11 02 AM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/e44c21cb-e8db-457e-9878-889fac3db7db)